Dockerfile

```diff
@@ -494,7 +512,7 @@ RUN mkdir -pv /rootfs/{boot,etc/cri/conf.d/hosts,lib/firmware,usr/local/share,us
 COPY --chmod=0644 hack/zoneinfo/Etc/UTC /rootfs/usr/share/zoneinfo/Etc/UTC
 RUN ln -s /usr/share/zoneinfo/Etc/UTC /rootfs/etc/localtime
 COPY --chmod=0644 hack/nfsmount.conf /rootfs/etc/nfsmount.conf
-RUN mkdir -pv /rootfs/{etc/kubernetes/manifests,etc/cni/net.d,usr/libexec/kubernetes}
+RUN mkdir -pv /rootfs/{etc/kubernetes/manifests,etc/cni/net.d,usr/libexec/kubernetes,extras,sysext}
```

Comment:
this was added to test the sysexts and extras PCR values appended to the initrd dynamically by sd-stub